pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -963,6 +963,16 @@ self: super: builtins.intersectAttrs super {
     '';
   }) super.jacinda;
 
+  nfc = overrideCabal (drv: {
+    isExecutable = true;
+    executableHaskellDepends = with self; drv.executableHaskellDepends or [] ++ [ base base16-bytestring bytestring ];
+    configureFlags = drv.configureFlags or [] ++ [ "-fbuild-examples" ];
+    postFixup = ''
+      ${drv.postFixup or ""}
+      mv $out/bin $doc/bin
```

Comment:
Why are you moving binaries to `$doc`? This will most certainly break assumptions in nixpkgs.

Better to set `enableSeparateBinOutput` to `true` and drop `postFixup` here.

Reply:
<img width="1202" alt="Screen Shot 2022-07-23 at 06 04 15" src="https://user-images.githubusercontent.com/543423/180600652-144c574d-b43e-4fc2-b985-ebd034b7218b.png">

Reply:
Since there are no binaries end-users would install nor would end users ever install this package directly (since it's just a library being pulled in), the `bin` output is free and probably easiest to use.